site/app/controllers/admin/PlagiarismController.php

```diff
@@ -864,8 +995,8 @@ public function configurePlagiarismForm(): WebResponse {
         $config["regex_dirs"] = ["submissions"];
         $config["language"] = array_fill_keys(PlagiarismUtils::getSupportedLanguages(), "");
         $config["language"]["plaintext"] = "selected";
-        $config["threshold"] = 5;
-        $config["sequence_length"] = 4;
+        $config["threshold"] = 15;
+        $config["sequence_length"] = 10;
```

Comment:
Can we have these values as class constants with an explanation for why they're the values they are, even if it's just a reference to look in another file.